internetarchive/openlibrary
# Use IdentifiersInput Vue component for work identifier UI

Closes #3430


Feature

### Technical

DRY implementation of work identifiers but currently unable to save the changes made by users into Solr. 

The work identifiers currently displayed in this component are based off of this list: https://openlibrary.org/config/work
### Testing

1. Navigate to a work/book and click on the edit button
2. Switch from the edition edit tab to the `Work Details` edit tab
![image](https://github.com/user-attachments/assets/f2d515fc-fb9e-4cde-ad25-4ade06a821c5)
3. Scroll to Work Identifiers section and add/remove work identifiers.
    - ~~Duplicates are currently not allowed for work identifiers. Adding a duplicate will warn the user that an identifier already exists.~~
    - ~~Work Identifiers should be saved as a list, allowing more than one ID per identifier. Duplicate ID numbers across different identifiers is not allowed i.e BookBrainz ID:123456 vs Amazon ID:123456.~~
    
![image](https://github.com/user-attachments/assets/0be7b550-9e53-48fa-896a-0fe2bf87e739)


4. After adding new work identifiers, save the changes and return to the editing page -> Work Details tab to check that the newly added work identifiers are still present.

### Stakeholders

@cdrini
